src/base/util_func.c

```diff
@@ -867,3 +873,64 @@ util_get_second_and_ms_since_epoch (time_t * secs, int *msec)
   assert (*msec < 1000);
 }
 // *INDENT-ON*
+
+/*
+ * util_is_localhost -
+ *
+ * return:
+ *
+ * NOTE:
+ */
+bool
+util_is_localhost (char *host)
```

Comment:
Since you moved the functions, it is also worthwhile to "fix" them. :)
```suggestion
util_is_localhost (const char *host)
```
because `host` is passed to a `const char*`

Reply:
Done.